public_html/dataView/allfiles.php

```diff
@@ -41,6 +41,8 @@
 	}
 
 	//determine the field name
+	// FIXME: this will only grab the LAST file field, if a form has Multiple
```

Comment:
there is absolutely nothing wrong with fixme and todos in code. They are reminders and notes. This code does not introduce a new issue. it is noting an existing issue that was discovered while reviewing code.

that existing issue is in no way related to the current problem. It should be addressed at a later date.